source/admin_manual/mailbox_formats/imapc.rst

```diff
@@ -10,3 +10,7 @@ optionally a more capable storage.
 
 For information on how to configure imapc in Dovecot, see
 :ref:`imapc_settings`.
+
+.. Note::
+
+   This mailbox format is not supported by Dovecot Pro.
```

Comment:
I think imapc has to be supported, since it's used for migrations and shared folders. @slusarz right?

Reply:
we can specify that it's supported for these usecases.